### PR TITLE
Histogram Plot UI: Limit number of buckets that user can choose (>0, <100)

### PR DESCRIPTION
**Pull request by KaroFr**

### CODE REVIEW

**Comment by KaroFr** [comment]
[Histogram Plot UI: Limit number of buckets that user can choose (>0, <n)](https://trello.com/c/4k0cEMrm/24-histogram-plot-ui-limit-number-of-buckets-that-user-can-choose-0-n)

File: projects/wave-core-new/src/lib/operators/dialogs/histogram-operator/histogram-operator.component.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ export class HistogramOperatorComponent implements OnInit, AfterViewInit, OnDest
                 },
             ),
             autoBuckets: [true, Validators.required],
-            numberOfBuckets: [20, Validators.required],
+            numberOfBuckets: [100, [Validators.required, Validators.min(0), Validators.max(100)]],
```
Comment (ChristianBeilschmidt) [inline projects/wave-core-new/src/lib/operators/dialogs/histogram-operator/histogram-operator.component.ts:86]: * the default should stay at 20
* I guess we need at least one bucket as minimum

File: projects/wave-core-new/src/lib/operators/dialogs/histogram-operator/histogram-operator.component.html [hunk 2]
```diff
@@ -45,6 +45,12 @@
         <div fxLayout="row" *ngIf="(form.controls.autoBuckets.valueChanges | async) === false">
             <mat-form-field fxFlex>
                 <input matInput type="number" placeholder="Number of Buckets" formControlName="numberOfBuckets" />
+                <mat-hint class="error" *ngIf="form.controls.numberOfBuckets.errors?.min || form.controls.numberOfBuckets.errors?.max">
+                    The number of buckets must be between 0 and 100.
```
Comment (ChristianBeilschmidt) [inline projects/wave-core-new/src/lib/operators/dialogs/histogram-operator/histogram-operator.component.html:49]: I guess we should define min and max in `histogram-operator.component.ts ` in order to ensure using the same value in the error message (template) and in the form definition.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: projects/wave-core-new/src/lib/operators/dialogs/histogram-operator/histogram-operator.component.ts (hunk 1 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/operators/dialogs/histogram-operator/histogram-operator.component.ts]: 1f0a88cacaa268afe16677b783518a944777aee1

File: projects/wave-core-new/src/lib/operators/dialogs/histogram-operator/histogram-operator.component.html (hunk 2 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/operators/dialogs/histogram-operator/histogram-operator.component.html]: 2d5bfc0ab5958370deb2a074152c5c6dc5ddef42
and 
ba9d0b2acf1262841474a4abd195afcd5ca59bdd

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**